ios-base/Info.plist

```diff
@@ -23,18 +23,24 @@
 	<key>CFBundleURLTypes</key>
 	<array>
 		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>fbXXXXXXXXXXX</string>
+				<string>fb703804262984729</string>
```

Comment:
I think we should remove this after testing. 
The idea is that if the client wants to use the feature, the assertion for a valid key would warn them. 
Also, the README is very clear about the use of facebook integration and I think it is pretty straight forward.